server/route/AppointmentController.js

```diff
@@ -169,4 +169,51 @@ router.put(
 	})
 );
 
+// Endpoint to get availability for each date in a specified month for a stylist
+router.get(
+	"/availability",
+	asyncHandler(async (req, res, next) => {
+		const { username, month } = req.body;
```

Comment:
Can you make this use GET query params instead of the body. See [req.query in express](https://stackoverflow.com/questions/6912584/how-to-get-get-query-string-variables-in-express-js-on-node-js) and [url params in GET requests](https://stackoverflow.com/questions/5767464/what-is-the-syntax-for-adding-a-get-parameter-to-a-url).